test/functional/JIT_Test/playlist.xml

```diff
@@ -747,7 +747,7 @@
 		echo $(Q)$(TEST_JDK_BIN)$(D)jitserver doesn't exist; assuming this JDK does not support JITServer and trivially passing the test.$(Q); \
 	fi; \
 	$(TEST_STATUS)</command>
-		<platformRequirements>os.linux,^arch.arm,^arch.aarch64,^arch.s390x,bits.64</platformRequirements>
+		<platformRequirements>os.linux,^arch.arm,^arch.aarch64,bits.64</platformRequirements>
```

Comment:
It would seem that the intent of #10876 was only to enable this test on Power, but because there is no `arch.s390x` architecture, it was also enabled for `arch.390`, so this change seems safe.